### DISCUSSION TITLE
Access to external service from odoo container

### DISCUSSION THREAD

**Discussion by rquevedo** [Tecnativa/doodba#471] [Q&A]
Hi folks,
I have a socks (This is not important, if is needed I change to http) proxy running on my host laptop. I need access from odoo python code to this proxy. Already try config with whitelist image and tcp-proxy image but without result.
Any help will be appreciated.

**Comment by pedrobaeza** [comment]
Do you mean in devel environment? You can switch temporarily `external: True` to False in the yaml.

**Reply by rquevedo** [reply]
Yes, is devel environment, but in my devel.yaml there is no section `external: True`. At end of file have:
`networks:
  default:
    internal: true
  public:`
Mi first attempt was set ` internal: false` but service running in my laptop is unavailable from odoo container. I test usisng `telnet host.docker.internal 9050`.

**Reply by pedrobaeza** [reply]
Sorry, it was `internal`, not external. The next step is to map your service, but that's something at Docker level, not Doodba specific.

**Reply by rquevedo** [reply]
ok, i will try. Thanks for the tip.

**Reply by bosd** [reply]
> ok, i will try. Thanks for the tip.


Hi @rquevedo How did you solve this?
I'm in a similar situation. Want to connect the odoo container to an service running on the host machine. (In devel env)

Currently Im on a empty developer db, but later on maybe a real db.
So my preference is to not expose the whole odoo network to the host, to prevent sending emails.
